issue_13/script.js

```diff
@@ -0,0 +1,86 @@
+const modalBtnWrapper = document.getElementById('js-modal-btn__wrapper');
+const modalBtn = document.getElementById('js-modal-btn');
+const modalWrapper = document.getElementById('js-modal__wrapper');
+const modalBack = document.getElementById('js-modal-back');
+const closeBtn = document.getElementById('js-close-btn');
+const requestBtn = document.getElementById('js-request-btn');
+const modalButtonElements = [modalBtn, modalBack, closeBtn];
+
+for (let i = 0; i < modalButtonElements.length; i++) {
+    modalButtonElements[i].addEventListener('click', () => {
+        modalBtnWrapper.classList.toggle('hide');
+        modalWrapper.classList.toggle('show');
+    });
+}
```

Comment:
I couldn't think of a way to do loop processing, so I learned it!

You can also assign a class and process it.
This is my opinion, so please consider it as a comment.

For example:
HTML : Add class "js-modal-btn" to button.
```
<body>
    <div class="modal-btn__wrapper" id="js-modal-btn__wrapper">
        <button type="button" class="modal-btn js-modal-btn">モーダルを表示する</button>
    </div>
    <div class="modal__wrapper" id="js-modal__wrapper">
        <div class="modal">
            <button type="button" class="request-btn" id="js-request-btn">リクエスト開始</button>
            <button type="button" class="close-btn js-modal-btn">
                <span class="close-btn__icon"></span>
            </button>
        </div>
        <button class="modal-back js-modal-btn"></button>
    </div>
    <div id="js-loading"></div>
    <ul id="js-ul"></ul>
</body>
```

JavaScript : Get className and do the loop process.
```
const modalBtnWrapper = document.getElementById('js-modal-btn__wrapper');
const modalWrapper = document.getElementById('js-modal__wrapper');
const requestBtn = document.getElementById('js-request-btn');
const modalButtonElements = document.getElementsByClassName('js-modal-btn');

for (let i = 0; i < modalButtonElements.length; i++) {
    modalButtonElements[i].addEventListener('click', () => {
        modalBtnWrapper.classList.toggle('hide');
        modalWrapper.classList.toggle('show');
    });
}
````

Reply:
Thanks for your comments!
I thought it would be easier to unify them in one class rather than giving each element a separate id for the same function, so I modified the method you suggested.
https://github.com/yoshikawa-akr/js-lessons/pull/13/commits/d06eca0f8b02d361f039c923a437a7d6476ff9dc